container_files/public_html/doc/builtin/Url.md

```diff
@@ -46,14 +46,12 @@ external tools.
 
 ## Credentials
 
-MLDB uses an subprogram known as `credentialsd` to keep credentials and supply them to the main
-MLDB process when required.
+MLDB can store credentials and supply them whenever required.
 
-This allows for MLDB users to control how credentials are stored and used, and
-to create temporary credentials for MLDB to limit its permissions to exactly what are needed.
+Capabilities have been added to control how credentials are stored and used, and
+to create temporary credentials to limit its permissions to exactly what are needed.
```

Comment:
This isn't true anymore.  With credentialsd, you could point MLDB to another external daemon which did this.  Now we no longer can, so we should remove this from the doc.